### PR TITLE
Fixed biot-savart scaling

### LINKED ISSUE #140: Biot-savart coil generation code generates in T/A, not Hz/A
Minor change needed to scale final output before returning fieldmap.

### PR DESCRIPTION
**Pull request by ghost**
Added a scaling term to translate output from T/A to Hz/A, as is standard. Fixes #140

### CODE REVIEW

**Review by jcohenadad — APPROVED** [review]
LGTM

File: test/test_biot_savart.py [hunk 1]
```diff
@@ -23,6 +25,8 @@ def test_normal_use(centers, normals, radii, segment_numbers, fov_min, fov_max,
     assert(basis[:, :, :, 0].shape == fov_n)
 
     # Check expected values
-    actual = np.round(basis[:, :, :, 0].reshape(len(axis_answers)), decimals=7)
-    expected = np.round(np.asarray(axis_answers), decimals=7)
+    actual = np.round(basis[:, :, :, 0].reshape(len(axis_answers)), decimals=-2)
+    expected = np.round(np.asarray(axis_answers), decimals=-2)
+    print(actual)
+    print(expected)
```
Comment (po09i) [inline test/test_biot_savart.py]: Once we have a debug flag I would print the output if we are in debug mode, for now I would remove.

File: test/test_biot_savart.py [hunk 2]
```diff
@@ -6,10 +6,12 @@
 
 from shimmingtoolbox.coils.biot_savart import biot_savart
 
+H_GYRO_R = 42.577478518e+6 # Hz/T
```
Comment (po09i) [inline test/test_biot_savart.py]: I would rename and update the value to be the same as the other file in this PR. This value has more decimal so I would update the other value.

**Review by po09i — APPROVED** [review]
LGTM

**Pull request merged**